build.jam

```diff
@@ -0,0 +1,37 @@
+# Copyright René Ferdinand Rivera Morell 2023-2024
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+constant boost_dependencies :
+    /boost/align//boost_align
+    /boost/assert//boost_assert
+    /boost/config//boost_config
+    /boost/container//boost_container
+    /boost/container_hash//boost_container_hash
+    /boost/core//boost_core
+    /boost/describe//boost_describe
+    /boost/endian//boost_endian
+    /boost/mp11//boost_mp11
+    /boost/static_assert//boost_static_assert
+    /boost/system//boost_system
+    /boost/throw_exception//boost_throw_exception ;
+
+project /boost/json
+    : common-requirements
+        <include>include
+    ;
+
+explicit
```

Comment:
Do all these targets have to be explicit? Currently the project builds as much as possible by default, and subprojects decide what parts of them have to be explicit.